src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -54,9 +55,11 @@ public AddCommand parse(String args) throws ParseException {
         ApplicationStatus applicationStatus = new ApplicationStatus(ApplicationStatus.PENDING_STATUS);
         InterviewStatus interviewStatus = new InterviewStatus(InterviewStatus.NOT_SCHEDULED);
         Availability availability = ParserUtil.parseAvailability(argMultimap.getValue(PREFIX_AVAILABILITY).get());
+        Remark remark = new Remark(""); // add command does not allow adding remarks straight away
```

Comment:
I believe the comment should be placed at the top from coding standard. Not sure! But either ways I don't have an issue with it. Just curious!